docs/src/how-to/install/configuration-options.rst

```diff
@@ -576,3 +576,308 @@ Second, also set the option under the `account-pages` section:
++------------------------------------+---------------------------------+------------------------------------+------------------------------------------+-------------------------------------------+----------------------------------+--------------------------------------+
+
+
+Changing the configuration on the server
+........................................
+
+To change the `searchSameTeamOnly` setting on the backend, edit the `values.yaml.gotmpl` file for the wire-server chart at this nested level of the configuration:
+
+.. code:: yaml
+
+  brig:
+    # ...
+    config:
+      # ...
+      optSettings:
+        # ...
+        setSearchSameTeamOnly: true
+
+If `setSearchSameTeamOnly` is set to `true` then `TeamSearchVisibility` is forced be in the `SearchVisibilityNoNameOutsideTeam` setting for all teams.
+
+Changing the default configuration for all teams
+................................................
+
+If `setSearchSameTeamOnly` is set to `false`  (or missing from the configuration) then the default value `TeamSearchVisibility` can be configured at this level of the configuration of the `value.yaml.gotmpl` file of the wire-server chart:
+
+
+.. code:: yaml
+
+  galley:
+    #...
+    config:
+      #...
+      settings:
+        #...
+        featureFlags:
+          #...
+          teamSearchVisibility: enabled-by-default
+
+This default value applies to all teams for which no explicit configuration of the `TeamSearchVisibility` has been set.
+
+
+Searching users on another (federated) backend
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+For federated search the table above does not apply, see following table.
+
+.. note::
+	
+	Incoming federated searches (i.e. searches from one backend to another) are considered always as being performed from a team user, even if they are performed from a personal user. 
+
+	This is because the incoming search request does not carry the information whether the user performing the search was in a team or not. 
+	
+	So we have to make one assumption, and we assume that they were in a team.
+
+Allowing search is done at the backend configuration level by the sysadmin:
+
+* Outbound search restrictions (`searchSameTeamOnly`, `TeamSearchVisibility`) do not apply to federated searches
+* A configuration setting `FederatedUserSearchPolicy` per federating domain with these possible values:
+  * `no_search` The federating backend is not allowed to search any users (either by exact handle or full-text).  
+  * `exact_handle_search` The federating backend may only search by exact handle
+  * `full_search` The federating backend may search users by full text search on display name and handle. The search search results are additionally affected by `SearchVisibilityInbound` setting of each team on the backend.
```

Comment:
those don't seem to get rendered to bullet point lists. Most likely these bullet point blobks need a blank line above them. There are other places in this doc with the same issue

Reply:
Fixed: 

![Screenshot from 2022-07-04 11-29-59](https://user-images.githubusercontent.com/108821/177126450-e34b14bd-9fc9-451e-9c28-44c23432170b.png)